cmd/server/common.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2019 The Moov Authors
+// Use of this source code is governed by an Apache License
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"crypto/rand"
+	"crypto/sha1"
+	"encoding/hex"
+)
+
+// NextID generates a new resource ID.
+// Do not assume anything about the data structure.
+//
+// Multiple calls to NextID() have no concern about producing
+// lexicographically ordered output.
+func NextID() string {
```

Comment:
Could we use `github.com/moov-io/base`'s `ID()` method? `NextID()` is an old version of that.

Reply:
Sure can, should an issue be created to change for ACH as well?